### PR TITLE
[eCW] fr_FR: Adjust enhancedMissingVehicles; Fix some translations.

### PR DESCRIPTION
**Pull request by fulax**
Since the fire support update in fr, some new vehicles were not counted as "driving".

Also fix some translated terms that are not appropriate.

### CODE REVIEW

**Comment by jxn-30** [comment]
Do you want to add/improve translations of other modules? Then you can do that all within that branch so we don't need a single PR for each module => less work 😉

**Comment by fulax** [comment]
Hi @jxn-30, Sure.

I have a habit of making small and consistent change sets tight a to a logical unit of code (here the module I guess). The contributing guide didn't define a policy about how large can a PR be, so I stuck to my habit.

If I add other translations, I'll make a single large PR. Here my first goal was to fix the missing vehicle table.

I can remove the translation part if you like.

Anyways, thanks for your feedback.

**Comment by jxn-30** [comment]
It's totaly fine to have just few changes in a PR but also one huge PR (for example when introducing a nee language and providing the full translation with one PR).

I just personally prefer the idea of having one PR for multiple changes within a short time but if you prefer having a PR for each module that's ok too.

**Comment by fulax** [comment]
Ok, let's wait a bit.
I'm working on the wiki part for this module, then I'll be done with this PR. Thanks again.

File: src/modules/extendedCallWindow/docs/fr_FR.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+![Véhicules requis](vehiculesManquants.png)
+
+### Aperçu des victimes
+Affiche dans l'en-tête de mission les véhicules de secours manquants entre ambulances, équipe médicales et hélicoptères de secours.
+
+![Besoins Victimes](apercuVictimes.png)
+
+### Temps de trajet le plus long
+Affiche, à côté du bouton « Déployer », le temps de trajet le plus long de l'ensemble des véhicules sélectionnés.
```
Comment (jxn-30) [inline src/modules/extendedCallWindow/docs/fr_FR.md:24]: Do you want to stick to the `« Déployer »` Method or use `` `Déployer` `` as we use it?
Our method produces such an output:

**Review by jxn-30 — COMMENTED** [review]
Looks fine to me.
Got a question about docs, but if you're ready we can merge :)

**Comment by fulax** [comment]
I think we're good now for that part. I had a friend re-read my french for grammar errors.

Thanks for the wiki tips.

**Review by jxn-30 — APPROVED** [review]

**Comment by jxn-30** [comment]
I merged but french docs are not yet built because there is no files in `/docs/fr_FR`. At least a readme is required.

**Comment by fulax** [comment]
Thank you ! The French Wiki home is my next huge task if I get bored

**Pull request merged**